airbyte-integrations/connectors/destination-pinecone/setup.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk",
+    "langchain",
+    "openai",
+    "requests",
+    "tiktoken",
+    "docarray[hnswlib]>=0.32.0",
+    "pinecone-client",
+    "typing-inspect==0.8.0",
+    "typing_extensions==4.5.0",
+    "pydantic==1.10.8",
+    "pandas==1.4.2",
+    "sentence-transformers",
```

Comment:
I don't think we need sentence-transformers anymore without the local hugging face models, right?